sdk/search/search/review/search.api.md

```diff
@@ -138,6 +138,18 @@ export interface CreateOrUpdateIndexOptions extends OperationOptions, ETagOperat
     allowIndexDowntime?: boolean;
 }
 
+// @public (undocumented)
```

Comment:
can you add ref docs for these? I was going to clean up the missing ones for index methods too

Reply:
Checked and confirmed no more undocumented is present.